docs/howto/initialize-database.rst

```diff
@@ -93,3 +95,7 @@ Initialize the database
 
     # initialize the database
     airflow db init
+
+.. spelling::
+
+     hba
```

Comment:
Can you revert it?

Reply:
Oh. We have second bug in this docs. Can you look at 76 line? It uses markdown syntax.

Reply:
I read it later. However, I am concerned about the error on line 76 now.

Reply:
I was thinking that a three character word was a higher risk of letting a typo somewhere else through -- so didn't want to allow it globally